src/status_im/hardwallet/core.cljs

```diff
@@ -974,6 +975,32 @@
                           (navigation/navigate-to-cofx :keycard-connection-lost nil))
                 (show-wrong-keycard-alert true)))))
 
+(fx/defn on-export-key-success
+  {:events [:hardwallet.callback/on-export-key-success]}
+  [{:keys [db] :as cofx} pubkey]
+  (let [multiaccount-address (get-in db [:multiaccount :address])
+        instance-uid (get-in db [:hardwallet :application-info :instance-uid])
+        callback-fn (get-in db [:hardwallet :on-export-success])
+        pairings (get-in db [:hardwallet :pairings])
+        event-to-dispatch (callback-fn pubkey)]
+    (re-frame/dispatch event-to-dispatch)
+    (fx/merge cofx
+              (clear-on-card-connected))))
+
+(fx/defn on-export-key-error
+  {:events [:hardwallet.callback/on-export-key-error]}
```

Comment:
use alias namespaced keywords instead of synthetic for events to prevent hidden circular dependencies

Reply:
Sorry, don't quite understand. If I use `::on-export-key-error` etc. instead, this will most likely require me to import `hardwallet.core` into `hardwallet.card` (if we want to use a convenient `::hardwallet/on-export-key-error` shorthand there), which will produce a circular dependency. Probably all `hardwallet.*` namespaces should be reorganized in this case.

Reply:
I think we can just get rid of `hardwallet.fx` ns and move all fx handlers from there into `hardwallet.core`. This will fix the circular dep problem. What do you think?

Reply:
Ok so there is already a circular dependency :D Well leave it like this then and create an issue this should be fixed at some point but not in this PR